src/ResourceManager/Profile/Commands.Profile/help/Resolve-AzureRmError.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.Azure.Commands.Profile.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Resolve-AzureRmError
+
+## SYNOPSIS
+
+Display detailed information about Azure PowerShell errors.
```

Comment:
No, we do special processing for azure powershell errors, but it is explained in the description that it applies to all errors as well